src/app.ts

```diff
@@ -18,17 +20,36 @@ const main = async () => {
 
   await dbConnection();
 
+  var bodyParser = require('body-parser');
+
+  var app = Express();
```

Comment:
any reason this was turned into a `var` from a `const`?

Reply:
honestly not entirely sure, this is what the stackoverflow post recommended, so i just followed along

Reply:
I'd make it a const